### PR TITLE
Add update settings tests

### PR DESCRIPTION
**Pull request by pscott**

### CODE REVIEW

File: starknet/src/tests/test_update_settings.cairo [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        assert(
+            space.min_voting_duration() == input.min_voting_duration,
+            'Min voting duration not updated'
+        );
+    // TODO: check event once it's been added
+    }
+
+    #[test]
+    #[available_gas(10000000000)]
+    // #[should_panic(expected: ('Invalid duration', 'ENTRYPOINT_FAILED'))] // TODO: uncomment once PR is merged
```
Comment (Orland0x) [inline starknet/src/tests/test_update_settings.cairo]: which PR is this?

Comment (pscott) [reply starknet/src/tests/test_update_settings.cairo]: #505 fixed the `update_settings` function

**Comment by Orland0x** [comment]
We need to actually store the daoURI in the state in order to be compatible with the standard like we do in sx-evm. I forgot to do this before
https://github.com/snapshot-labs/sx-evm/blob/df37a70673067300a3a6c0541740117a86d0bc59/src/Space.sol#L49C12-L49C12

**Comment by pscott** [comment]
> We need to actually store the daoURI in the state in order to be compatible with the standard like we do in sx-evm. I forgot to do this before https://github.com/snapshot-labs/sx-evm/blob/df37a70673067300a3a6c0541740117a86d0bc59/src/Space.sol#L49C12-L49C12

Yup, funny you mention it, I'm on it with [this PR](https://github.com/snapshot-labs/sx-starknet/pull/529), maybe we can merge this one now so I re-update the `update_settings` function?

**Comment by Orland0x** [comment]
> > We need to actually store the daoURI in the state in order to be compatible with the standard like we do in sx-evm. I forgot to do this before https://github.com/snapshot-labs/sx-evm/blob/df37a70673067300a3a6c0541740117a86d0bc59/src/Space.sol#L49C12-L49C12
> 
> Yup, funny you mention it, I'm on it with [this PR](https://github.com/snapshot-labs/sx-starknet/pull/529), maybe we can merge this one now so I re-update the `update_settings` function?

yup cool

**Review by Orland0x — APPROVED** [review]
uTACK

**Pull request merged**